app/Http/Controllers/PostController.php

```diff
@@ -3,10 +3,15 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
 
 class PostController extends Controller
 {
-   public function index(){
-       return view('posts.index');
+    public function index(){
```

Comment:
```suggestion
    public function index() 
    {
```
Bracket in functie altijd op volgende regel